server/websockets.py

```diff
@@ -45,3 +44,12 @@ async def notify_user_list_to_client(client: str, users: List[str]):
             'users': users,
         },
     )
+
+
+async def notify_end_game_to_client(players: List[str], data: Dict):
+    for client in players:
+        await manager.send(
+            client,
+            websocket_events.EVENT_END_GAME,
+            data,
+        )
```

Comment:
Is there a way to do this in bulk? because one of this notifications could fail and none of the players will know

Reply:
It should/will have an error handler in manager.send to avoid this. We could spawn a separate task for each notification but the error handler is needed anyway.

Reply:
I'm saying that you could modify the `manager.send` to send several notifications at once and be able to retry in case one notification fails. This behaviour could be used for several use cases, so it's better to have it in one place